basic/src/test/java/org/jboss/hal/testsuite/test/configuration/transactions/JournalTestCase.java

```diff
@@ -2,32 +2,25 @@
 
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.hal.testsuite.category.Shared;
-import org.jboss.hal.testsuite.dmr.Composite;
 import org.jboss.hal.testsuite.dmr.Operation;
+import org.jboss.hal.testsuite.util.Console;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
 
 import java.io.IOException;
+import java.util.concurrent.TimeoutException;
 
-import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.NAME;
-import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.VALUE;
-import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.WRITE_ATTRIBUTE_OPERATION;
-
-/**
- * @author Jan Kasik <jkasik@redhat.com>
- *         Created on 12.10.15.
- */
 @RunWith(Arquillian.class)
 @Category(Shared.class)
-//Some test might fail due to the HAL-883
 public class JournalTestCase extends TransactionsTestCaseAbstract {
 
-    //TODO prepare for JournalStore Config
     @BeforeClass
-    public static void setUp() {
+    public static void setUp() throws InterruptedException, TimeoutException, IOException {
         prepareForJournalConfiguration();
     }
```

Comment:
The @BeforeClass method should be directly the `prepareForJournalConfiguration` as there is nothing else done and it is good to name methods based on what they are really doing, not just encapsulate single method call (I don't see any benefit in it.)

But OK.